ansible/roles/install-microshift/defaults/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# install-microshift default vars
+
+enable_repos:
```

Comment:
Why do we need RHEL 9 repos for MicroShift? 
Should we just build RPMs here and install them?
See https://github.com/openshift/microshift/blob/main/docs/devenv_rhel8.md#build-microshift

Reply:
I have created these scripts based upon installation of MicroShift from the COPR repo. 
Though even with building the RPMs, would the build not need prerequisite packages to be installed from the repos?

Reply:
COPR packages are outdated. We should probably check it it for the first pass and then discuss our next steps. We need support for RHEL8 as all the development and runtime is now on this OS.

Reply:
Agreed

Reply:
I've added support for building and installing the MicroShift RPMs from source on RHEL8.